commit 72dad5cc7b731e94e76bab8d91d9cc6fd089961c (HEAD -> master)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-09-27T08:14:18-07:00 (10 days ago)
    
    TRITON-1907 Updating docker when not present gives the wrong install command
    Reviewed by: Pedro Palazón Candel <pedro@joyent.com>
    Approved by: Pedro Palazón Candel <pedro@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index d30a227..af4792f 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -10,6 +10,10 @@
 
 # sdcadm Changelog
 
+## 1.30.3
+
+- TRITON-1907 output the correct command to install the docker service
+
 ## 1.30.2
 
 - TRITON-1696 sdcadm should verify that triton_cns_enabled is set for the admin
diff --git a/lib/procedures/update-dockerlogger.js b/lib/procedures/update-dockerlogger.js
index 2c5538e..4be3f3c 100644
--- a/lib/procedures/update-dockerlogger.js
+++ b/lib/procedures/update-dockerlogger.js
@@ -137,7 +137,7 @@ UpdateDockerlogger.prototype.execute = function udExecute(opts, callback) {
                     } else if (!service) {
                         next(new UpdateError(
                             'docker service does not exist. Please run:\n' +
-                            '\n    sdcadm experimental update-docker\n' +
+                            '\n    sdcadm post-setup docker\n' +
                             '\n\nbefore trying to install dockerlogger'));
                     } else {
                         next();
diff --git a/package.json b/package.json
index 5f36d1b..23adc34 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdcadm",
   "description": "Administer a Triton Data Center",
-  "version": "1.30.2",
+  "version": "1.30.3",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
